src/scss/includes/components/poiItem.scss

```diff
@@ -0,0 +1,33 @@
+.poiItem {
+  padding: 16px 30px;
+  cursor: pointer;
+  display: flex;
+
+  @media (max-width: 640px) {
+    padding: 12px 16px;
+  }
+
+  .poiTitleImage {
+    width: 65px;
+    height: 65px;
+  }
+
+  &-address {
+    text-overflow: ellipsis;
+    max-width: 210px;
```

Comment:
Trick: You can avoid this fixed value and have something really responsive by giving the rule `min-width: 0;` to the container `div` (unnamed for now, but could be `.poiItem-left`)

Reply:
Ok, I added a `left` class. Not sure about the benefits of setting `min-width: 0` through

Reply:
It's required for the ellipsis to work without the hard-coded size. As I understand it, the idea is this rule forces the browser to compute an explicit size for the container during its layout phase.
I'll test it on multiple browsers to see if it does the trick everywhere.